tests/entrypoints/openai/test_structured_output.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: Apache-2.0
+from enum import Enum
+from typing import NamedTuple
+
+import openai  # use the official client for correctness check
+import pytest
+import pytest_asyncio
+from pydantic import BaseModel, ValidationError
+
+from ...utils import RemoteOpenAIServer
+
+# # any model with a chat template should work here
+MODEL_NAME = "Qwen/Qwen2.5-3B-Instruct"
```

Comment:
can we use a smaller models?

also we have a structured outputs tests for this check v1/entrypoints

Reply:
Done.